### DISCUSSION TITLE
Default configuration for lsp-mode sideline diagnostics

### DISCUSSION THREAD

**Discussion by niontrix** [syl20bnr/spacemacs#16280] [Q&A] [answered]
Hi folks,

I was wondering if it would make sense to set the default `lsp-ui-sideline-diagnostic-max-lines` in the lsp-layer to something else than 1 (which is the current default), e.g. 5 or something like that. Because more often than not the diagnostic text has line breaks in it, which then pretty much invalidates the purpose of using the sideline diagnostics in the first place.
What do you think about that?

KR

**Comment by smile13241324** [comment]
Not a bad idea, prs are welcome

**Comment by smile13241324** [comment, marked as answer]
Done in latest develop
